### ISSUE TITLE
Version 0.10.0 cannot load gif with ANT build

### ISSUE THREAD

**Issue by s011208** [facebook/fresco#1236]
```
05-24 11:16:46.931: W/app(30167): java.lang.NullPointerException: Attempt to invoke interface method 'com.facebook.imagepipeline.image.CloseableImage com.facebook.imagepipeline.animated.factory.AnimatedImageFactory.decodeGif(com.facebook.imagepipeline.image.EncodedImage, com.facebook.imagepipeline.common.ImageDecodeOptions, android.graphics.Bitmap$Config)' on a null object reference
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.decoder.ImageDecoder.decodeGif(ImageDecoder.java:113)
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.decoder.ImageDecoder.decodeImage(ImageDecoder.java:88)
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.producers.DecodeProducer$ProgressiveDecoder.access$200(DecodeProducer.java:2189)
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.producers.DecodeProducer$ProgressiveDecoder$1.run(DecodeProducer.java:129)
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.producers.JobScheduler.access$000(JobScheduler.java:1207)
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.producers.JobScheduler$1.run(JobScheduler.java:78)
05-24 11:16:46.931: W/app(30167):   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
05-24 11:16:46.931: W/app(30167):   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
05-24 11:16:46.931: W/app(30167):   at com.facebook.imagepipeline.core.PriorityThreadFactory$1.run(PriorityThreadFactory.java:43)
05-24 11:16:46.931: W/app(30167):   at java.lang.Thread.run(Thread.java:818)
```

By the way, I use ANT build and I think that animated-base and animated-gif have been included in project because I can declare class (such as com.facebook.animated.gif.GifFrame frame = null;) in animated-gif without build failed

**Comment by tyronen** [comment]
We don't actually support building with Ant. How did you set this up?

**Comment by s011208** [comment]
It is hard to tell you how we set the build process because we have modified part of ant build environments. However, are there any tips that I can do to figure out what I missed? I have tried to load library "gitimage" and declare class in animated-gif. It seems that both of methods are working great. 

By the way, I have included following folders "animated-base", "animated-gif", "drawee", "fbcore", "fresco", "imagepipeline" and "imagepipeline-base".
